requirements/generated/requirements-dev.txt

```diff
@@ -24,7 +24,7 @@ click==8.1.7
     # via pip-tools
 distlib==0.3.8
     # via virtualenv
-django==5.0.6
+django==4.2.13
```

Comment:
Did we mean to downgrade this...?
Perhaps our pinning in the .in file needs to specify at least 5 as well as less than 5.1?

Reply:
nope - now resolved